taxcalc/params.json

```diff
@@ -920,9 +920,9 @@
     },
     "_NIIT_trt":{
         "long_name": "Net Investment Income Tax rate (UIMC)",
-	    "description": "AKA Unearned Income Medicare Distribution Tax. Taxed on net investment income of individuals with MAGI higher than certain thresholds.",
-        "irs_ref": "Form , line , ",
-        "notes": "HC",
+	"description": "Also called Unearned Income Medicare Contribution Tax. You will be taxed at this rate if your MAGI is higher than NIIT threshold.",
```

Comment:
If I remember correctly, the tax rate is applied to the entire investment income rather than the portion above the threshold, I'll check again.